### ISSUE TITLE
Incorrect JSON body for removing items from playlists

### ISSUE THREAD

**Issue by Fluxanoia** [spotify-api/spotify-api.js#38]
As per the [docs](https://developer.spotify.com/documentation/web-api/reference/#endpoint-remove-tracks-playlist), when removing tracks from a playlist, URIs should be wrapped in an object. However, I believe the [current state of the code](https://github.com/spotify-api/spotify-api.js/blob/3b77f609e7fd106c0c0ab45e5b13904466205662/src/managers/PlaylistManager.ts#L286) misses this.

I'm using this code to remove items, which is almost identical to my code for adding items (which is working):

```
export const removeFromPlaylist = async (
    playlist : Playlist,
    uris : Array<SpotifyURI>,
    throwError : (error : Error) => void,
) => {
    let offset = 0;
    do {
        const result = await playlist.remove(uris.slice(offset, offset + spotifyTrackRemoveLimit))
                                     .catch(throwError);
        if (!result) return false;
        offset += spotifyTrackRemoveLimit;
    } while (offset < uris.length);
    return true;
};
```

I'm attaching stack traces and whatnot, but I think I've pretty much already diagnosed the problem:

![image](https://user-images.githubusercontent.com/6279354/124008174-0d0a3900-d9d4-11eb-9077-534c03d1514d.png)
![image](https://user-images.githubusercontent.com/6279354/124008652-8f92f880-d9d4-11eb-855b-68e0ab99bf44.png)

**Comment by Scientific-Guy** [comment]
I am sorry i pretty much could not understand your problem but you mean that `Playlist.removeItems` is not working? It seems you have received `400` request can you show the response sent by the api.

**Comment by Fluxanoia** [comment]
The [Spotify API documentation](https://developer.spotify.com/documentation/web-api/reference/#endpoint-remove-tracks-playlist) specifies that when deleting tracks from a playlist, the 'tracks' sent to the API should be URIs wrapped in an object:

![image](https://user-images.githubusercontent.com/6279354/124092578-eccb9000-da4e-11eb-9c1b-70843b72143b.png)

But as it stands, spotify-api.js just comma separates the URIs (seen [here](https://github.com/spotify-api/spotify-api.js/blob/3b77f609e7fd106c0c0ab45e5b13904466205662/src/managers/PlaylistManager.ts#L286), snippet below):

```
body: {
    ...opts,
    tracks: items.join(',')
} as RawObject
```

I attached some sample code to my issue, to illustrate where I found this issue and also just in case I was making a silly mistake. The API response is: `400: JSON body doesn't conform to specification`.

**Comment by Scientific-Guy** [comment]
Well yes. I did not notice that. Thanks for the submission of issue. This will be fixed on the upcomming bug fix release `v8.1.1`.

**Comment by Scientific-Guy** [comment]
This bug has been cleared in the release `v8.1.1`. Further bug reports can be submitted in a new issue.
